blocks/carousel/carousel.js

```diff
@@ -104,7 +104,7 @@ function calculateSlideHeight(carousel, slide) {
       const bodyHeight = parseFloat(bodyStyle.lineHeight) * lineCount;
       const figureStyle = window.getComputedStyle(slide.querySelector('figure'));
       const figureHeight = figureStyle ? parseFloat(figureStyle.height) : SLIDE_CAPTION_SIZE;
-      carousel.style.height = `${bodyHeight + figureHeight + 32}px`;
+      carousel.style.height = `${bodyHeight + figureHeight + 132}px`;
```

Comment:
We need to go back and test this change for when the "full width" carousel variation is used.